.github/auto_assign.yml

```diff
@@ -1,28 +0,0 @@
-# Set to true to add reviewers to pull requests
```

Comment:
Isn't this something that we want to keep? We don't want to not auto assign reviewers